cmds/cmd_functions/functions.js

```diff
@@ -4,13 +4,17 @@ const _ = require('lodash');
 const fs = require('fs-extra');
 const archiver = require('archiver');
 const Promise = require('bluebird');
-const reply = require('./reply')();
 const path = require('path');
+const reply = require('./reply');
 
 module.exports = (tjmConfig) => {
-    let terasliceClient = require('teraslice-client-js')({
-        host: `${tjmConfig.cluster}`
-    });
+    let terasliceClient;
+
+    if (!tjmConfig.cluster) {
+        terasliceClient = require('teraslice-client-js')({
+            host: tjmConfig.cluster
+        });
+    }
 
```

Comment:
tjmConfig should always have a cluster specified at this point.  I'm not sure what this is trying to do, a client is required here.

Reply:
everything else looks good.